### ISSUE TITLE
Can I use this grunt plugin to DOWNLOAD a file from a remote server ?

### ISSUE THREAD

**Issue by FrancoAguilera** [spmjs/grunt-scp#33]
I see on the documentation : 

> Copy files to remote server

But is it possible to use it to download a file from the server ?.

**Comment by dkurucz** [comment]
Good idea!  I'll look into supporting this.  Will keep this issue open for the time being...
